examples/maze-solver/maze.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2019 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import dwavebinarycsp
+import re
+
+
+def build_bqm(n_rows, n_cols, start, end, walls, penalty_per_tile=0.5):
```

Comment:
It might be worth renaming this `maze_to_bqm` or similar

Reply:
I shall rename it as `get_maze_bqm(..)` as "get" will imply that something is returned.